corehq/sql_db/util.py

```diff
@@ -3,7 +3,7 @@
 from __future__ import unicode_literals
 import uuid
 from collections import defaultdict
-from numpy import random
+import random
```

Comment:
This breaks `select_db_for_read(weighted_dbs)` on line 296,
```python
        return random.choice(dbs, p=normalized_weights)
```
because random.choice doesn't take a parameter `p`. :disappointed:

Reply:
(If this is the only place we use numpy, we could roll our own weighted choice.)

Reply:
@dannyroberts  I think it makes sense to remove the numpy removal from this PR in that case and it can be handled in norman's PR

Reply:
Oh sweet! Okay, I'll revert the removal of numpy in this PR